app/src/functions/notify-endpoint-usage/app.ts

```diff
@@ -1,31 +1,36 @@
 import { influx, buildAppUsageQuery } from "../../lib/influx";
 import { getUTCTimestamp, getHoursFromNowUtcDate } from "../../lib/date-utils";
 import connect from "../../lib/db"
-import { ApplicationData, ExtendedApplicationData, ExtendedLoadBalancerData, GetUsageDataQuery } from "../../models/types";
+import { ApplicationData, ExtendedApplicationData, ExtendedLoadBalancer, ExtendedLoadBalancerData, GetUsageDataQuery } from "../../models/types";
 import User from '../../models/User'
-import { getApplicationNetworkData, getAppsInNetwork } from "../../lib/pocket";
-import { QueryAppResponse, Application } from '@pokt-network/pocket-js';
+import { getAppsInNetwork } from "../../lib/pocket";
+import { Application } from '@pokt-network/pocket-js';
 import ApplicationModel, { IApplication } from "../../models/Application";
 import LoadBalancerModel, { ILoadBalancer } from "../../models/LoadBalancer";
+import Redis from 'ioredis'
 import { retryEvery } from "../../utils/retry";
 
 const redisHost = process.env.REDIS_HOST || "";
 const redisPort = process.env.REDIS_PORT || "";
 
+const cacheTTL = parseInt(process.env.NETWORK_CACHE_TTL ?? '') || 3600;
+const queryStartTime = parseInt(process.env.INFLUX_QUERY_START_TIME ?? '') || 1;
 const maxRetries = process.env.MAX_RETRIES || 3;
```

Comment:
I know you guys don't do it on the portal API, but this is something I think it's worth doing. Let's capitalize these so it's obvious they're constants.